google-cloud-translate/src/main/java/com/google/cloud/translate/spi/v2/HttpTranslateRpc.java

```diff
@@ -126,6 +126,7 @@ private GenericUrl buildTargetUrl(String path) {
               .setSource(sourceLanguage)
               .setKey(options.getApiKey())
               .set("model", Option.MODEL.getString(optionMap))
```

Comment:
@mziccard @garrettjonesgoogle was there a reason to use the `set` method here instead of `setModel` (in https://github.com/GoogleCloudPlatform/google-cloud-java/commit/60270261fa763131f6cd6b9aacfc7819e0af1c3d#diff-1bf2355a417917946c22e0783005adf1R130 )?